src/SystemServicesContext/useIsLocationOn.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState, useCallback } from "react"
+import { Platform } from "react-native"
+
+import useOnAppStateChange from "./useOnAppStateChange"
+import { isLocationEnabled } from "../gaen/nativeModule"
+
+const useIsLocationOn = (): boolean => {
```

Comment:
Could this be a bit more specific, like determine the location for android or something like that? Would that be relevant or make the structure easier to understand? Naming with much detail is deceiving some times but maybe it makes sense here? Would that contribute to the reason why we split it or would help the split?

Reply:
That's an interesting point. In this case, I think it makes sense to leave the hook name platform agnostic because the platform check is handled within the hook as opposed to being handled outside in the context. What do you think of that?